librz/core/cmd_tasks.c

```diff
@@ -40,7 +41,10 @@ static int task_break(RzCore *core, int tid) {
 		eprintf ("This command is disabled in sandbox mode\n");
 		return -1;
 	}
-	if (tid) {
+	if (!tid) {
```

Comment:
What about using `rz_return_val_if_fail()` instead?

Reply:
It's not an assertion-level failure, the user could have passed 0, but that is the main task which should not be breaked.